cucumber/step_definitions/screenshot.js

```diff
@@ -0,0 +1,16 @@
+const { After, Status } = require("cucumber");
```

Comment:
Could you please create a support folder, and move this file to the folder? It contains hooks, you can rename it to hooks.js. 
If you use this script in case of missing screenshots folder, the script will not work, it will return with error. Could you please create a screenshots folder and push it to repository?

Reply:
done